src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -4598,6 +4622,8 @@ else if (serverMajorVersion >= 9) // Yukon (9.0) --> TDS 7.2 // Prelogin disconn
             writeFedAuthFeatureRequest(true, tdsWriter, fedAuthFeatureExtensionData);
         }
 
+        writeUTF8SupportFeatureRequest(true, tdsWriter);
```

Comment:
Length is being "calculated" and returned, but we don't use the value. Seems like we're always using half of the functionalities.